phenopy/util.py

```diff
@@ -45,16 +60,18 @@ def export_phenotype_hpoa_with_no_parents(phenotype_hpoa_file, phenotype_hpoa_no
         exit(1)
 
     no_parents_df = df.copy()
-    for gene, annotations in df.groupby('#DatabaseID'):
-        termlist = [node for node in annotations['HPO_ID'].tolist() if node in hpo_network.nodes()]
+    for gene, annotations in df.groupby("database_id"):
```

Comment:
hmm looks like these are being updated because of the header changes too. I am start wondering if the intention is to only work with the new version of hpo file (with new header) or we still want to have both the old version and new version work in phenopy?

Reply:
made the necessary changes in `util.py` to account for both old and new versions.